web/src/utilities/utiltyFunctions.ts

```diff
@@ -49,3 +51,56 @@ export function opacityForZoomLevel(currentZoom: number, maxZoom: number): numbe
   }
   return maxOpacity - (0.4 * diff);
 }
+
+export function centroidStringToCoords(input: string): Array<number> {
+  const centroidString = input.match(/\(([0-9|.|\s])+\)/);
+  if (!centroidString) return [-1, -1];
+  const centroid = centroidString[0].slice(1, -1).split(' ').map(
+    (val: string) => parseFloat(val),
+  );
+  return centroid;
+}
+
+export interface NucleusGlandDistance{
+  nucleus: number | undefined;
+  gland: number | undefined;
+  line: Array<Array<number>>;
```

Comment:
Is this always a 2x2 array? Any thoughts on using `Point[]` as the type?

Reply:
Changed in [39c875f](https://github.com/atlascope/atlascope/pull/193/commits/39c875f4db286a66f66b76328637181902f1ce1c)